coolkit/dependencies/dependency.py

```diff
@@ -55,12 +43,28 @@ def is_installed(soft):
     return False
 
 
+def get_available_package_manager(dependency_map):
```

Comment:
rename it to `_get_supported_package_managers`

I use undersore as prefix for those methods that are only for this script particularly. I wont be calling them from outside in case I import this script into another.

Reply:
now you will think of  other such functions like print_err and print_clr or line_adder. these are my standard functions i use them frequently in many files in many projects. so I keep some standard functions

Reply:
dont worry much about naming of other functions that I haven't commented, I will comment in case some change is required.